brainstormforce/astra
# AST-2045 - Updated old header-footer layout's svg's in radio image control

### Description
- As in custom redesign update we redesign our controls
- Due to this legacy radio image control's UI broke

### Screenshots
- https://share.bsf.io/geuRKEjX

### Types of changes
- Bug fix (UI change)

### How has this been tested?
- Use following filter to switch to old header footer layout
- Verify the result over there
```
add_filter( 'astra_is_header_footer_builder_active', '__return_false' );
```
- Check following cases
- 1. Header sections (primary header, above header, below header)
- 2. Footer sections (footer widgets, footer bar)

### Checklist:
